adapters/IronSource/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # ironSource Adapter for Google Mobile Ads SDK for iOS
 
-## 6.7.5.0
+
+## 2.6.1
+- Initiate IronSource SDK once per ad unit
+
+## 2.6.0
```

Comment:
We want to keep 6.7.5.0. This represents the 6.7.5.0 version of the adapter we have hosted on https://bintray.com/google/mobile-ads-adapters-ios/IronSourceAdapter#files/IronSourceAdapter/6.7.5.0, which was built against 6.7.5.0 of the IronSource SDK. Release notes here should add a 6.7.7.0 that talks about initiating IronSource SDK once per ad unit.

Reply:
Sure ,  Updated the Change log accordingly.